CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ option(JSRUNTIMEHOST_CORE_SCRIPTLOADER "Include Babylon Native Core ScriptLoader
 option(JSRUNTIMEHOST_POLYFILL_CONSOLE "Include Babylon Native Polyfill Console." ON)
 option(JSRUNTIMEHOST_POLYFILL_SCHEDULING "Include Babylon Native Polyfill Scheduling." ON)
 option(JSRUNTIMEHOST_POLYFILL_XMLHTTPREQUEST "Include Babylon Native Polyfill XMLHttpRequest." ON)
+option(JSRUNTIMEHOST_POLYFILL_ABORTCONTROLLER "Include Babylon Native Polyfill AbortController." ON)
+option(JSRUNTIMEHOST_POLYFILL_ABORTSIGNAL "Include Babylon Native Polyfill AbortSignal." ON)
```

Comment:
I think this should be one polyfill. I would not expect one to be available without the other.